src/main/java/frc/robot/commands/groups/ScoreCube.java

```diff
@@ -1,35 +1,25 @@
 package frc.robot.commands.groups;
 
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
-import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
 import frc.robot.Constants;
-import frc.robot.commands.drive.SetDriveRotationTarget;
 import frc.robot.commands.floorArm.FloorMoveArmToPostion;
 import frc.robot.commands.intake.floor.FloorDrop;
-import frc.robot.subsystems.ArmSubsystem;
 import frc.robot.subsystems.DrivetrainSubsystem;
 import frc.robot.subsystems.FloorArmSubsystem;
 import frc.robot.subsystems.FloorIntake;
 
-public class RotateAndScoreCube extends SequentialCommandGroup {
+public class ScoreCube extends SequentialCommandGroup {
 
-  public RotateAndScoreCube(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, ArmSubsystem arm, FloorIntake intake) {
+  public ScoreCube(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, FloorIntake intake) {
     addCommands(
-      new SetDriveRotationTarget(drive, 180),
-      new WaitUntilCommand(() -> Math.abs(drive.getRotation().getDegrees() - 180) < 5),
-      new SetDriveRotationTarget(drive), 
-
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_SHOOTING_POSITION),
       new WaitCommand(0.5),
       new FloorDrop(intake, Constants.FLOOR_INTAKE_FAST_SPEED).withTimeout(1),
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT_POSITION),
-
-      new SetDriveRotationTarget(drive, 0),
-      new WaitUntilCommand(() -> Math.abs(drive.getRotation().getDegrees()) < 5),
-      new SetDriveRotationTarget(drive), 
-      new WaitCommand(0.5)
-      
+      new WaitCommand(0.5),
+      new InstantCommand(drive::zeroGyroscope, drive)
```

Comment:
We will need to resolve #49 as well. Based on my prior testing our pose will be thrown off when we zero the gyro. Its a small change so easy to implement in this PR as well.

I did a little research on chiefdelphi and found that some teams tie the auto initial pose to resetPose and when resetPose is called they reset the gyro and offset its angle accordingly. Same concept when zeroing the gyro. 

```
        // replaces zeroGyroscope()
        public void zeroRotation() {
                Pose2d pose = new Pose2d(getPose().getTranslation(), Rotation2d.fromDegrees(0.0));
                resetPose(pose);
        }

        public void resetPose(Pose2d pose) {
                navx.reset();
                navx.setAngleAdjustment(pose.getRotation().getDegrees());
                simRotation = pose.getRotation().getDegrees();
                odometry.resetPosition(
                        getGyroscopeRotation(),
                        getModulePositions(), 
                        pose
                );
        }
```

If this is implemented and we use path planner to define the initial robot rotation (via holonomic angle) it's not required to zero the gyroscope after scoring the cube. The orientation would be assumed based on initial auto position from PathPlanner. I think this would be a nice generic way of handling this.